POC/ColorPicker/main.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <windows.h>
+#include "Utility.h"
+
+int main()
+{
+    HDC window = GetWindowDC(NULL);
+    while (true)
+    {
+        POINT cursorPosition;
+        if (GetCursorPos(&cursorPosition))
+        {
+            COLORREF color = GetPixel(window, cursorPosition.x, cursorPosition.y);
+            std::cout << "X: " << cursorPosition.x << "Y: " << cursorPosition.y << '\n';
+            Utility::printColorRef(color);
+        }
+    }
```

Comment:
is CTRL+C only way of exiting?

Reply:
Yes, that's a good point. We made this proof of concept to be a very simple quick demo to show how the tech with the Win32 API calls works, not as a shippable product. As we iterate on it, we'll end up removing this entirely as we'll be using a UI front-end. So we didn't bother with extra usability sections yet, as this code is sure to change and these features would be lost. We're making this PR just to validate our usage of the API and work from here by building up progressive MVPs. Do you think this is the right way to do it?

Reply:
Yep, agree that these are first iteration changes to have a demo going; was curious to know if usability was a consideration yet - appears to be as you continue to build towards MVP